DirectProgramming/DPC++/N-BodyMethods/Nbody/README.md

```diff
@@ -0,0 +1,88 @@
+# Nbody sample
+An N-body simulation is a simulation of a dynamical system of particles, usually under the influence of physical forces, such as gravity. This nbody sample code is implemented using C++ and DPC++ language for Intel CPU and GPU. 
+  
+| Optimized for                       | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04; Windows 10
+| Hardware                          | Skylake with GEN9 or newer
+| Software                          | Intel&reg; oneAPI DPC++ Compiler beta;
+| What you will learn               | How to offload the computation to GPU using Intel DPC++ compiler
+| Time to complete                  | 15 minutes
+
+## Purpose
+Nbody sample code simulates 16000 particles and for 10 integration steps. Each particles position, velocity and acceleration parameters are dependent on other (N-1) particles. This algorithm is highly data parallel and a perfect candidate to offload to GPU. The code demonstartes how to deal with multiple device kernels which can be enqueued into a DPC++ queue for execution. 
+
+## Key Implementation Details 
+The basic DPC++ implementation explained in the code includes device selector, buffer, accessor, kernel, and command groups.
+
+## License  
+This code sample is licensed under MIT license. 
+
+## Building the `Nbody` Program for CPU and GPU
+
```

Comment:
Please add comment to Readme in regards to location of dpc_common.hpp, should look like this:

Building the Program for CPU and GPU
Include Files
The include folder is located at %ONEAPI_ROOT%\dev-utilities\latest\include on your development system.

as seen in https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/sepia-filter

Reply:
Readme is updated in the new version